### PR TITLE
IPAM configure and show-borrowed/configuration

### PR DESCRIPTION
**Pull request by gianlucam76**
## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 10ba082060ff9ee9c034d830e5123ee68fab1f14

https://deploy-preview-3471--calico.netlify.app

File: reference/calicoctl/ipam/configure.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: calicoctl ipam
```
Comment (caseydavenport) [inline reference/calicoctl/ipam/configure.md:2]: "calicoctl ipam configure" ?

File: reference/calicoctl/ipam/show.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Options:
   -h --help             Show this screen.
      --ip=<IP>          Report whether this specific IP address is in use.
      --show-blocks      Show detailed information for IP blocks as well as pools.
+     --show-borrowed    Show detailed information for "borrowed" IP addresses.
+     --configuration    Show current Calico IPAM configuration
```
Comment (caseydavenport) [inline reference/calicoctl/ipam/show.md:25]: Hm, didn't spot this in code review - should probably be `--show-configuration` to be symmetric with the other options. 

Should be a quick fix.

File: reference/calicoctl/ipam/configure.md (hunk 1 above)
Comment (gianlucam76) [reply reference/calicoctl/ipam/configure.md:2]: show and release had same title. So I changed all three of them to "calicoctl ipam show" "calicoctl ipam release" "calicoctl ipam configure"

**Pull request merged**